firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsController.java

```diff
@@ -782,9 +777,10 @@ File getNativeSessionFilesDir() {
 
   private Task<Void> logAnalyticsAppExceptionEvent(long timestamp) {
     if (firebaseCrashExists()) {
-      Logger.getLogger().d("Skipping logging Crashlytics event to Firebase, FirebaseCrash exists");
+      Logger.getLogger().v("Skipping logging Crashlytics event to Firebase, FirebaseCrash exists");
```

Comment:
I might keep this as DEBUG just so it is more likely we'll find it in the rare case it causes a problem for a customer. I don't feel super strongly about that though.

Reply:
Might give this one a bit more thought, there might be a way we can make this less noisy and more useful in a DEBUG log.